pyppeteer/coverage.py

```diff
@@ -277,14 +284,16 @@ def _onExecutionContextsCleared(self, event: Dict) -> None:
         if not self._enabled:
             raise PageError('CSSCoverage is not enabled.')
         self._enabled = False
-        result = await self._client.send('CSS.stopRuleUsageTracking')
-        await self._client.send('CSS.disable')
-        await self._client.send('DOM.disable')
+        ruleTrackingResponse = await self._client.send('CSS.stopRuleUsageTracking')
+        await asyncio.gather(
+            self._client.send('CSS.disable'),
+            self._client.send('DOM.disable')
+        )
         helper.removeEventListeners(self._eventListeners)
 
         # aggregate by styleSheetId
         styleSheetIdToCoverage: Dict = {}
```

Comment:
superflous type annotation
```suggestion
        styleSheetIdToCoverage = {}
```